test/gerrit_test.py

```diff
@@ -1,6 +1,5 @@
 from oci import gerrit
 
-
```

Comment:
This is change is unrelated and unwanted (pep8 violation), but I can remove it while merging.